transform/run_pipelines.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o pipefail
+set -o nounset
+# Uncomment this line to see each command for debugging (careful: this will show secrets!)
+# set -o xtrace
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+FACTORY_NAME=$(cd "${SCRIPT_DIR}"/../infrastructure/transform && terragrunt output adf_name | tr -d '"')
+RESOURCE_GROUP_NAME=$(cd "${SCRIPT_DIR}"/../infrastructure/core && terragrunt output core_rg_name | tr -d '"')
+export FACTORY_NAME
+export RESOURCE_GROUP_NAME
+
+run_pipeline_and_wait () {
+    local pipeline_name="${1}"
```

Comment:
should we validate that there's a value passed into the script (and exit early if not)?

Reply:
We could, personally I don't think it's really worth doing as this function is called in the same file, and there's only one argument.

Reply:
Where is {1} coming from? I'd assumed it was from outside this script

Reply:
It's coming from line 58

Reply:
Cool